source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -395,11 +395,24 @@ that a Node developer would have trouble reading your code is advised against.
 
 ## Frameworks
 
-> Use Express for web applications, avoid lesser-known frameworks
+It is important to remember that we use the
```

Comment:
This was not part of the original set of comments, but I felt it was important to highlight as this requirement informs the choices that are available.